.github/workflows/deploy.yml

```diff
@@ -0,0 +1,94 @@
+name: CI/CD Pipeline
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
```

Comment:
Now, on local env we using node version 18?

Reply:
For now we use `pnpm v9` and it require node at least v18.

Reply:
If we use node version as same local env, I think it will be streamlines more line use minimum require version, to prevert version issuse in future. Is it possible to change?

Reply:
sure for now it compatible for 18 and 20. I will change it to 20.